lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -480,7 +480,7 @@ class Distribution(object):
     OS_FAMILY_MAP = {'RedHat': ['RedHat', 'Fedora', 'CentOS', 'Scientific', 'SLC',
                                 'Ascendos', 'CloudLinux', 'PSBM', 'OracleLinux', 'OVS',
                                 'OEL', 'Amazon', 'Virtuozzo', 'XenServer', 'Alibaba',
-                                'EulerOS', 'openEuler', 'AlmaLinux'],
+                                'EulerOS', 'openEuler', 'AlmaLinux', 'RockyLinux'],
```

Comment:
Please change to 'Rocky' -- The ID is simply `Rocky`
```suggestion
                                'EulerOS', 'openEuler', 'AlmaLinux', 'Rocky'],
```